aip/0132.md

```diff
@@ -80,7 +81,7 @@ message ListBooksRequest {
   - The comment above the message **should** document the maximum allowed
```

Comment:
"above the message" or "above the field" ?

Reply:
Done.